data/group/src/main/java/com/kappzzang/jeongsan/di/GroupRetrofitModule.kt

```diff
@@ -0,0 +1,30 @@
+package com.kappzzang.jeongsan.di
+
+import com.kappzzang.jeongsan.api.GroupRetrofitService
+import com.kappzzang.jeongsan.build_config.BuildConfig
+import com.kappzzang.jeongsan.datasource.remote.GroupRemoteDataSource
+import dagger.Module
+import dagger.Provides
+import dagger.hilt.InstallIn
+import dagger.hilt.components.SingletonComponent
+import javax.inject.Singleton
+import retrofit2.Retrofit
+import retrofit2.converter.gson.GsonConverterFactory
+
+@Module
+@InstallIn(SingletonComponent::class)
+object GroupRetrofitModule {
+
+    @Provides
+    @Singleton
+    fun provideGroupRemoteDataSource(groupApi: GroupRetrofitService) =
+        GroupRemoteDataSource(groupApi)
+
+    @Provides
+    @Singleton
+    fun provideGroupRetrofitService(): GroupRetrofitService = Retrofit.Builder()
+        .baseUrl(BuildConfig.SERVICE_URL)
+        .addConverterFactory(GsonConverterFactory.create())
+        .build()
+        .create(GroupRetrofitService::class.java)
```

Comment:
P1: `:common:retrofit` 모듈에 ServiceRetrofit을 미리 정의해 두었기 때문에, 그걸 활용하면 아래와 같이 변경할 수 있을 것 같아요!

 ```suggestion
    @Provides
    @Singleton
    fun provideGroupRetrofitService(
        @ServiceRetrofit serviceRetrofit: Retrofit
    ): GroupRetrofitService = serviceRetrofit.create(GroupRetrofitService::class.java)
```